content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -233,7 +233,7 @@ spec:
       istioNamespace: external-istiod
       meshID: mesh1
       multiCluster:
-        clusterName: remote_cluster
+        clusterName: $CTX_REMOTE_CLUSTER
```

Comment:
Think this won't work because it is the context, not the cluster name.  If you want cluster name as var, user would need to set them as env var first.

Reply:
Good point. Added another env var. PTAL